samples/nrf9160/at_client/README.rst

```diff
@@ -0,0 +1,142 @@
+
+
+Requirements
+************
+
+* The following development board:
+
+  * nRF9160 DK board (PCA10090)
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/at_client`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+
+Connecting to the nRF9160 DK board with LTE Link Monitor
+========================================================
+
+To connect to the nRF9160 DK board with LTE Link Monitor, perform the following steps:
+
+#. Launch the LTE Link Monitor after programming the AT Client sample onto the nRF9160 DK board.
+#. Connect the nRF9160 DK board to the PC with USB.
+#. Power on the nRF9160 DK board.
+#. Click :command:`Select Device` and select the particular board device entry from the drop-down list in the LTE Link Monitor.
+#. Observe that initially the command :command:`AT+CFUN?` is automatically sent to the modem, which returns a value 4, indicating that the modem is in the offline mode.
+#. Observe that the LTE Link Monitor terminal display also shows :command:`AT+CFUN=1` indicating that the modem has changed to the normal mode.
+#. Perform a reset of the nRF9160 DK board by pressing the reset button, which will reboot the board and start the AT Client sample.
+
+
+Testing
+=======
+
+After programming the sample to your board, test the sample by performing the following steps:
+
+#. `Connect to the nRF9160 development board with LTE Link Monitor <Connecting to the nRF9160 DK board with LTE Link Monitor_>`_.
+#. Run the following commands from the LTE Link Monitor terminal:
+
+   * Enter the command :command:`AT+CFUN?`.
+
+     This command connects to an LTE network and triggers the command :command:`AT+CFUN=1` which will change the modem state to the normal mode.
+
+   * Enter the command :command:`AT+CFUN?` into the LTE Link Monitor terminal again.
+
+     The UART/Modem/UICC/LTE/PDN indicators in the LTE Link Monitor side panel turn green.
+     This command also automatically launches a series of commands like:
+
+     * :command:`AT+CGSN=1` which displays the product serial idenitification number(IMEI).
+     * :command:`AT+CGMI`   which displays the manufacturer name.
+     * :command:`AT+CGMM`   which displays the model identification name.
+     * :command:`AT+CGMR`   which displays the revision identification.
+     * :command:`AT+CEMODE` which displays the current mode of operation.
+
+   * Enter the command :command:`AT+CLAC`.
+
+     This command displays a list of all the available AT commands.
+
+
+   * Enter the command :command:`AT%XOPERID`.
+
+     This command returns the network operator ID.
+
+
+   * Enter the command :command:`AT%XMONITOR`.
+
+     This command returns the modem parameters.
+
+
+   * Enter the command :command:`AT%XTEMP?`.
+
+     This command displays the current modem temperature.
+
+
+
+
+Sample output
+=============
+
+The following image shows the sample output of the command :command:`AT+CLAC`:
+
+.. figure:: /images/at_clac.PNG
```

Comment:
I would prefer adding the output of the command as text and not as an image. The output will be different anyway when using the Link Monitor app or a console.

Reply:
Replaced the image with the sample output text.